가운데-글자-가져오기.js

```diff
@@ -0,0 +1,17 @@
+function solution(s) {
+    const dex = parseInt(s.length / 2);
+    let answer = 0;
+    if(s.length % 2 === 1) {
+        return answer = [dex];
+    }else if(s.length %2 === 0) {
+        return answer = [dex] + [dex + 1];
+    }
```

Comment:
```suggestion
    if(s.length % 2 === 1) {
        return [dex];
    }else {
        return [dex] + [dex + 1];
    }
```
입력 문자열 s 의 길이는 항상 자연수 이기 때문에 2로 나눈 나머지는 항상 1 과 0 입니다.
if조건식의 여집합인 else 를 사용해도 돌아가고,
return 을 만나면 함수가 끝나기 때문에 else 를 쓰지않고 반환만 해도 정상적으로 돌아 갈 것 같습니다.

굳이 answer 에 담지않고 바로 반환도 가능합니다